### PR TITLE
LSP: Readme, changelog, and version modified for 12.4 Beta.

### PR DESCRIPTION
**Pull request by dbalek**
LSP: Readme, changelog, and version modified for 12.4 Beta.

### CODE REVIEW

File: java/java.lsp.server/vscode/CHANGELOG.md [hunk 1]
```diff
@@ -20,22 +20,37 @@
...
 
 -->
+
+## Version 12.4 Beta
+
+* Test results UI added using Test Explorer UI extension
+* Micronaut projects support added
+  * YAML configuration files code completion
+* Maven multi-project projects support
+* Maven repository index download is not performed
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/CHANGELOG.md:30]: I don't think this is so great achievement to be mentioned in a change log.

File: java/java.lsp.server/vscode/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	"description": "Apache NetBeans Language Server Extension for Visual Studio Code",
 	"author": "Apache NetBeans",
 	"license": "Apache 2.0",
-	"version": "12.3.0",
+	"version": "12.3.9",
```
Comment (JaroslavTulach) [inline java/java.lsp.server/vscode/package.json:7]: What does this version means?

**Review by sdedic — APPROVED** [review]

File: java/java.lsp.server/vscode/package.json (hunk 2 above)
Comment (MartinBalin) [reply java/java.lsp.server/vscode/package.json:7]: This means 12.4 Beta as it was discussed on dev@nb mailing list. It is shorter than proposed 12.3.990 yet still will serve the purpose.

File: java/java.lsp.server/vscode/CHANGELOG.md (hunk 1 above)
Comment (dbalek) [reply java/java.lsp.server/vscode/CHANGELOG.md]: Removed.

File: java/java.lsp.server/vscode/package.json (hunk 2 above)
Comment (dbalek) [reply java/java.lsp.server/vscode/package.json:7]: The last version before 12.4.0 ? Hard to assign a reasonable version number for 12.4 Beta...

File: java/java.lsp.server/vscode/README.md [hunk 3]
```diff
@@ -43,8 +45,12 @@ to set your environment up to support
 
 * __Java: Compile Workspace__ - invoke Maven or Gradle build
 * __GraalVM: Pause in Script__ - place a breakpoint into first executed polyglot script
-* debugger __Java 8+__ - start test or main class on JDK8+ in polyglot mode
+* Debugger __Java 8+__ - start test or main class on JDK8+ in polyglot mode
 * Progress shown for long running operations with cancel support for selected types
+* __Native Image Debugger__ is a new Run configuration addede which allows Java style debugging of Ahead of Time compiled native-images, produced by GraalVM. It is experimental feature whichworks with GDB on Linux. GDB 7.11 or GDB 10.1 is required due to known issue [#26139](https://sourceware.org/bugzilla/show_bug.cgi?id=26139) in GDB 8 and 9.
```
Comment (entlicher) [inline java/java.lsp.server/vscode/README.md:50]: `a new Run configuration addede` - there's an extra `e`.
`whichworks` - a missing space.

Comment (dbalek) [reply java/java.lsp.server/vscode/README.md]: Fixed.

**Pull request merged**